index.js

```diff
@@ -62,22 +76,27 @@ conventionalRecommendedBump({
     console.error(chalk.red(err.message))
     return
   }
-
   var newVersion = pkg.version
   if (!argv.firstRelease) {
-    newVersion = semver.inc(pkg.version, release.releaseAs)
+    newVersion = getNextVersion(release)
     checkpoint('bumping version in package.json from %s to %s', [pkg.version, newVersion])
     pkg.version = newVersion
     fs.writeFileSync(pkgPath, JSON.stringify(pkg, null, 2) + '\n', 'utf-8')
   } else {
     checkpoint('skip version bump on first release', [], chalk.red(figures.cross))
   }
-
-  outputChangelog(argv, function () {
+  // only commit the package.json on prerelease
```

Comment:
@stvnwrgs, I think I'm with @stevemao. Let's generate a CHANGELOG for pre-releases, with this caveat:

It would be neat if there was eventually logic that could once `v1.0.0` is released, combine the information from  `v1.0.2.alpha.0`, `v1.0.3.alpha.1` into a single `v1.0.0` entry in the CHANGELOG; I would start simple though and not have a special case for pre-release handling.